helm/flink-kubernetes-operator/templates/rbac.yaml

```diff
@@ -89,6 +81,7 @@ rules:
       - update
       - patch
       - delete
+      - deletecollection
```

Comment:
Why are we adding the delete collection verb to many places? Is this used somewhere?

Reply:
I just added them for "completeness". Because in the past the verbs just was a star ("*"), a `deletecollection` is implicitly a part of it. If not needed, I can also remove it from this PR.